resources/revision.rb

```diff
@@ -322,16 +322,22 @@ def before_delete(&block)
 
 # Method returning a list of paths for all current releases
 # @since 1.0.0
+# Amended to prioritize validating the current folder's revision
+# @since 1.1.0
 def current_release_paths
-  ::Dir.glob(::File.join(deploy_to, '/releases/*')).sort
+  paths_array = Array.new
+  paths_array.push(if ::File.exist?(current_path)
+                   then ::File.realpath(current_path)
+                   end)
```

Comment:
otherwise when not ::File.exist? , you're pushing nil into an array for no reason, just so you can ` flatten.compact  ` below to get rid of nil